app/src/main/java/com/amazonaws/kinesisvideo/demoapp/fragment/StreamWebRtcConfigurationFragment.java

```diff
@@ -191,10 +186,20 @@ public void onClick(final View view) {
     }
 
     private void startMasterActivity() {
+        final SparseBooleanArray checked = mOptions.getCheckedItemPositions();
+        
+        // Check if Send Video is selected (position 0)
+        if (!checked.get(0)) {
+            new AlertDialog.Builder(getActivity())
+                    .setPositiveButton("OK", null)
+                    .setMessage("Send Video must be enabled to start as master!")
```

Comment:
For master P2P, they can do audio-only (video not required) or none

Only in master ingestion, video + audio required